.changes/v0.27.0.md

```diff
@@ -0,0 +1,13 @@
+## v0.27.0 (2025-03-29)
+
```

Comment:
please add a notice section to explain that future extension version upgrade will be backward compatible (e.g prepare for 1.0 release, etc).